app/todolist.js

```diff
@@ -5,8 +5,10 @@
     this.items = itemsApi.items;
     this.item = {};
 
-    this.addItem = function(item) {
-      itemsApi.addItem(item);
+    this.addItem = function(itemDescription) {
+      if(description) {
```

Comment:
Yes, you're right, in fact, i didn't see that the app it doesn't working hehe! I will going to make a last PR (i'm hope so). After, tell me what do you think about the early return modify.